R/inference.R

```diff
@@ -52,17 +61,25 @@ rater <- function(data,
                   model,
                   method = "mcmc",
                   data_format = "long",
+                  long_data_names_spec = list(
```

Comment:
I suggest changing `long_data_names_spec` to `long_data_colnames` everywhere

Reply:
Good idea! Will do